pkg/operator/yugabytedb/controller.go

```diff
@@ -340,10 +373,19 @@ func createMasterContainerCommand(namespace, serviceName string, masterCompleteN
 		fmt.Sprintf("--replication_factor=%d", replicas),
 		"--logtostderr",
 	}
+
+	if &resources.Limits != nil {
+		memLimit, ok := getMemoryLimitBytes(resources)
```

Comment:
nit: in general it's preferred to check for ok on the same line, such as the following.
```
if memLimit, ok := getMemoryLimitBytes(resources); ok {
...
}
```

Or if on a separate line, no need for a blank line between. Not critical though if you prefer it this way.

Reply:
Updated